erratatool.yml

```diff
@@ -154,6 +154,25 @@ boilerplates:
 
       All OpenShift Container Platform 4.12 users are advised to upgrade to these updated packages and images when they are available in the appropriate release channel. To check for available updates, use the OpenShift CLI (oc) or web console. Instructions for upgrading a cluster are available at https://docs.openshift.com/container-platform/4.12/updating/updating-cluster-cli.html
     solution: *common_solution
+  microshift:
+    synopsis: MicroShift RPM
+    topic: |
+      Red Hat build of MicroShift release 4.12.1 is now available with updates to packages and images that fix several bugs.
+    description: |
+      Red Hat build of MicroShift is Red Hat's light-weight Kubernetes orchestration solution designed for edge device deployments and is built from the edge capabilities of Red Hat OpenShift. MicroShift is an application that is deployed on top of Red Hat Enterprise Linux devices at the edge, providing an efficient way to operate single-node clusters in these low-resource environments.
+
+      This advisory contains the RPM packages for Red Hat build of MicroShift 4.12.1. See the following advisory for the container images for this release:
```

Comment:
Is there a way to have a variable for the version number in the text? I see "4.12.z" on line 144, for example, but I don't know if that text gets substituted automatically or if someone has to fix it by hand.

Reply:
We fix that by hand. Unfortunately, there isn't a way to automate that.

Reply:
Ack.